metashare/api/jobs.py

```diff
@@ -88,15 +38,16 @@ def try_to_make_branch(repository, *, new_branch, base_branch):
                 raise
 
 
-def create_branches_on_github(*, user, repo_url, project, task, repo_root):
+def _create_branches_on_github(*, user, repo_url, project, task, repo_root):
     """
     Expects to be called in the context of a local github checkout.
     """
-    gh = login(token=user.gh_token)
+    gh = gh_given_user(user)
```

Comment:
@wlonk It looks like currently `user.gh_token` and `user.github_account` are not being used. We can probably use them (within `gh_given_user`) or just remove them?